### PR TITLE
gh-83270: Update IDLE's credits

### LINKED ISSUE #83270: Update IDLE's credits
BPO | [39089](https://bugs.python.org/issue39089)
--- | :---
Nosy | @terryjreedy, @taleinat

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = 'https://github.com/terryjreedy'
closed_at = None
created_at = <Date 2019-12-18.17:44:49.810>
labels = ['expert-IDLE', '3.7', '3.8', '3.9']
title = "Update IDLE's credits"
updated_at = <Date 2020-04-24.20:45:04.897>
user = 'https://github.com/taleinat'
```

bugs.python.org fields:
```python
activity = <Date 2020-04-24.20:45:04.897>
actor = 'taleinat'
assignee = 'terry.reedy'
closed = False
closed_date = None
closer = None
components = ['IDLE']
creation = <Date 2019-12-18.17:44:49.810>
creator = 'taleinat'
dependencies = []
files = []
hgrepos = []
issue_num = 39089
keywords = []
message_count = 3.0
messages = ['358636', '365296', '367222']
nosy_count = 2.0
nosy_names = ['terry.reedy', 'taleinat']
pr_nums = []
priority = 'normal'
resolution = None
stage = 'needs patch'
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue39089'
versions = ['Python 3.7', 'Python 3.8', 'Python 3.9']
```

</p></details>

### PR DESCRIPTION
**Pull request by erlend-aasland**

### CODE REVIEW

**Comment by erlend-aasland** [comment]
Tal, Jan: I'm not sure this is what you envisioned. Please feel free to make any changes; I just took what I found in the issue and tried to put it into the current CREDITS.txt.

**Comment by erlend-aasland** [comment]
I'm not sure a NEWS entry is needed here. I'll happily apply one if you disagree, though.

File: Lib/idlelib/CREDITS.txt [hunk 1]
```diff
@@ -28,6 +29,15 @@ Jim Jewett, Martin v. Löwis, Jason Orendorff, Guilherme Polo, Josh Robb,
 Nigel Rowe, Bruce Sherwood, Jeff Shute, and Weeble have submitted useful
 patches.  Thanks, guys!
 
+Major contributors since 2005:
+
+- 2005: Tal Einat
+- 2010: Terry Jan Reedy (current maintainer)
+- 2013: Roger Serwys and GSOC Phil Webster
```
Comment (terryjreedy) [inline Lib/idlelib/CREDITS.txt]:
```suggestion
- 2013: Roger Serwys
```

File: Lib/idlelib/CREDITS.txt [hunk 2]
```diff
@@ -28,6 +29,15 @@ Jim Jewett, Martin v. Löwis, Jason Orendorff, Guilherme Polo, Josh Robb,
 Nigel Rowe, Bruce Sherwood, Jeff Shute, and Weeble have submitted useful
 patches.  Thanks, guys!
 
+Major contributors since 2005:
+
+- 2005: Tal Einat
+- 2010: Terry Jan Reedy (current maintainer)
+- 2013: Roger Serwys
```
Comment (terryjreedy) [inline Lib/idlelib/CREDITS.txt:36]:
```suggestion
- 2013: Roger Serwy
```

**Review by terryjreedy — APPROVED** [review]

**Pull request merged**